inst/doc/geoknife.R

```diff
@@ -4,6 +4,9 @@ options(continue=" ")
 options(width=60)
 library(knitr)
 library(geoknife)
+query <- geoknife::query
+`values<-` <- geoknife::`values<-`
+id <- geoknife::id
```

Comment:
gotchya. you think it's worth placing these re-definitions right after you load each of those other packages, along with a quick explanation? seems like something a geoknife user might need to know about explicitly for future use.